src/Compilers/VisualBasic/Portable/Symbols/Source/SourcePropertySymbol.vb

```diff
@@ -205,7 +205,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
                         prop,
                         Binder.GetAccessorName(prop.Name, MethodKind.PropertySet,
                                                isWinMd:=prop.IsCompilationOutputWinMdObj()),
-                        flags Or SourceMemberFlags.MethodKindPropertySet Or SourceMemberFlags.MethodIsSub,
+                        (flags Or SourceMemberFlags.MethodKindPropertySet Or SourceMemberFlags.MethodIsSub) And Not SourceMemberFlags.Iterator,
```

Comment:
Would it be better to change SourcePropertyAccessorSymbol's constructor to erase the flag so that callers wouldn't have to worry about it?